File: mvpbase/src/main/java/com/lcm/android/mvp/BaseMvpPresenter.java
package com.lcm.android.mvp;



/**
 * ****************************************************************
 * Author: LCM
 * Date: 2017/5/22 下午12:10
 * Desc:
 * *****************************************************************
 */

public class BaseMvpPresenter<V extends BaseView> implements BasePresenter<V>{
    private V mMvpView;

    @Override
    public void onAttachView(V view) {
        mMvpView = view;
    }

    @Override
    public void onDetachView() {
        mMvpView = null;
    }

    public V getMvpView() {
        return mMvpView;
    }


}

File: mvpbase/src/main/java/com/lcm/android/mvp/BaseMvpPresenter.kt
package com.lcm.android.mvp


/**
 * ****************************************************************
 * Author: LCM
 * Date: 2017/5/22 下午12:10
 * Desc:
 * *****************************************************************
 */

open class BaseMvpPresenter<V : BaseView> : BasePresenter<V> {
    var mvpView: V? = null
        private set

    override fun onAttachView(view: V) {
        mvpView = view
    }

    override fun onDetachView() {
        mvpView = null
    }


}
